test/cases/testdata/sets/test-sets-1077.yaml

```diff
@@ -0,0 +1,103 @@
+cases:
+- data:
```

Comment:
There's some history to this, but in new files, it would improve readability if we removed all the `data` that is not used in the actual case below.

Also, the number in the file name is no longer relevant. You could call it `test-sets-multiply.yaml`, for example.